clang/tools/binary-symbol-scanner/BinarySymbolScanner.py

```diff
@@ -0,0 +1,446 @@
+#--------------------------------------------------------------------------------
+# MIT License
+# 
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Year info is not required, and to avoid update the year info after updating the script, suggest remove the year info.

Reply:
Done!